document-service/doc-api/pyproject.toml

```diff
@@ -0,0 +1,181 @@
+[tool.poetry]
+name = "doc-api"
+version = "0.1.0"
+description = ""
+authors = ["dlovett <doug@daxiom.com>"]
+license = "Apache 2.0"
+readme = "README.md"
+packages = [{include = "doc_api", from = "src"}]
+
+[tool.poetry.dependencies]
+python = "^3.12"
+flask = "^3.0.3"
+flask-sqlalchemy = "^3.1.1"
+Flask-Cors = "^4.0.1"
+flask-migrate = "^4.0.7"
+Flask-Pydantic = "^0.12.0"
+python-dotenv = "^1.0.1"
+gunicorn = "^22.0.0"
+pycountry = "^24.6.1"
+email-validator= "1.3.1"
+phonenumbers = "8.13.39"
+notifications-python-client = "7.0.0"
+beautifulsoup4 = "4.12.3"
+sqlalchemy-continuum = "^1.4.2"
+pg8000 = "^1.31.2"
+cachecontrol = "^0.14.0"
+structured-logging = { git = "https://github.com/bcgov/sbc-connect-common.git", subdirectory = "python/structured-logging", branch = "main" }
+oauthlib = "^3.2.2"
+Jinja2 = "^3.1.2"
+google-api-core = "^2.11.0"
+google-auth-oauthlib = "^1.0.0"
+google-auth = "^2.16.2"
+google-cloud-core = "^2.3.2"
+google-cloud-pubsub = "^2.15.0"
+google-cloud-storage = "^2.7.0"
+googleapis-common-protos = "^1.58.0"
+grpc-google-iam-v1 = "^0.12.6"
+grpcio-status = "^1.51.3"
+pytz = "^2022.7.1"
+alembic-utils = "^0.8.3"
+datedelta = "^1.4"
+flask-jwt-oidc = "^0.7.0"
+
+[tool.poetry.group.test.dependencies]
+psycopg2-binary = "^2.9.9"
+pytest = "^8.2.2"
+pytest-cov = "^5.0.0"
+pytest-mock = "^3.14.0"
+requests-mock = "^1.12.1"
+faker = "^25.9.2"
+freezegun = "^1.5.1"
+
+[tool.poetry.group.dev.dependencies]
+black = "^24.4.2"
+pylint = "^3.2.5"
+bandit = "^1.7.9"
+flake8-pyproject = "^1.2.3"
+isort = "^5.13.2"
```

Comment:
we're moving to zimports (used by SQLAlchemy) and has a better fit for Flask & SQLAlchemy

Reply:
Thanks, I'll replace isort with zimports in the next PR.